city_scrapers/settings/__init__.py

```diff
@@ -0,0 +1,15 @@
+import logging
+import os
+
+# https://jansonh.github.io/scrapinghub-gcs/
+# If "GOOGLE_APPLICATION_CREDENTIALS" is the credential rather than a path,
+# we need to write a local file with the credential.
+path = "{}/google-cloud-storage-credentials.json".format(os.getcwd())
+
+credentials_content = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
```

Comment:
Can we use `getenv` here so that this doesn't throw an error when not supplied? It's probably not as much of an issue if it's commented out unless GCS is used, but might make sense to play it safe

Reply:
Done, good call.